status-im/status-react
# feat: select asset screen ui

fixes #16914 

### Summary

This PR integrates asset selection page UI, using fetched data from status-go and filtering through search input

![](https://github.com/status-im/status-mobile/assets/18485527/685a3242-fc3d-4b03-8449-6e8fbb8fb86f)
![](https://github.com/status-im/status-mobile/assets/18485527/1163dc0c-fe75-4f8b-8daa-a18d679c8584)

#### Platforms

- Android
- iOS

##### Functional

- wallet / transactions


### Steps to test

- Open Status
- Open new wallet
- Select an account
- Tap on Send
- enter an address and tap Continue
- Check asset selection page

status: ready
